docs/Analog.md

```diff
@@ -1,32 +1,83 @@
 # AMDC Analog Input
 
-This document describes the design considerations and implementation details for the analog input subsystem on the Advanced Motor Drive Controller (AMDC).
+This document describes the design considerations and implementation details for the analog input subsystem on the Advanced Motor Drive Controller (AMDC). A block diagram is presented and each component discussed in detail. Specifications of each component are provided based on the datasheet.
+
+## Relevant Versions of AMDC Hardware
+
+AMDC REV D
 
 ## Design Requirements and Considerations
 
-Discussion related to inherient requirements and why those requirements exist.
-- Low noise for ICs
+There are several design requirements that must be met for the analog signal chain of AMDC:
+
+1. Synchronously digitize differential analog inputs (bipolar, ±10 V at each input of the differential signal).
+2. Operate ADC fast enough such that sampling can be aligned to PWM carrier waveform (>100ksps throughput on all channels with accurate timing).
+3. High impedance inputs so no possiblity for ground loops.
+4. High common-mode rejection ratio (CMRR) for wide range of inputs.
+5. Low noise signal chain.
+
+As it was mentioned, the analog input to AMDC should be bipolar differential with the voltage within ±10 V. Then, the analog front-end converts this into pseudo-differential bipolar which is needed by the analog-to-digital converter (ADC). More information on ADC input types and their definitions can be found [here](https://www.analog.com/media/en/technical-documentation/product-selector-card/2PB_sarinputtypesfb.pdf).
 
 ## Block Diagram / External Connections
 
-Pictorial representation of analog system, with blocks and descriptions of what they do.
+<img src="images/amdc-analog.svg" />
 
-## Schematics
+### Analog connectors
+Analog signal chain of AMDC can receive up to 8 bipolar differential analog inputs (16 signals in total – 8 positive and 8 negative). There are 4 analog connectors (modular jacks) each receiving up to 2 differential inputs. There are also 4 ESD protection chips to protect sensitive electronics from damage due to ESD. 
 
-Discussion related to schematics for analog subsystem:
-- Related screenshots
-- Hierarchy
+A table of the pin mappings for each analog connector is shown below:
+
+| Pin number | Signal name |
+|------------|--------|
+| 1 | Analog P1 |
+| 2 | Analog N1 |
+| 3 | +15 V |
+| 4 | GND |
+| 5 | -15 V |
+| 6 | GND |
+| 7 | Analog P2 |
+| 8 | Analog N2 |

```

Comment:
Writing style check. In other documents we have [inconsistently] labeled signals with a block like: `Analog P1`. I am not requesting that you switch to using that as much as that you have a consistent style for labeling these.